Java/cp/Longest_Pallindromicsubstring.java

```diff
@@ -0,0 +1,78 @@
+
+// This is a Java program  to find the longest palindromicsubstring from a word using dynamic programming approach
```

Comment:
Go through some merged PR to get a better idea of how to write a beginner pitch of a program.